src/build.js

```diff
@@ -11,6 +12,8 @@ import { setCwd } from './plugins/npm-plugin/registry.js';
 export default async function build(options = {}) {
 	options.out = options.out || 'dist';
 
+	await premove(options.out, { cwd: options.cwd });
```

Comment:
nit: I think this needs to happen after the `normalizeOptions()` call on line 20.

Reply:
Replacing my comment for this. Didn't scroll down 🤦‍♂️